stylesheets/site-structure.scss

```diff
@@ -90,7 +90,7 @@ h1.panel-title {
 }
 
 .screenshot > a > img {
-    max-height: 480px;
+    max-height: 640px;
```

Comment:
I don't like scrolling so much, which IIRC is why this was set to 480px. If an image is bigger, you can click on it to view the full size image.